mix.exs

```diff
@@ -106,4 +109,25 @@ defmodule Hexpm.MixProject do
       end
     end)
   end
+
+  defp vendor_hex_erl(_) do
```

Comment:
@ericmj I think the confusion was because I have incorrectly copy-pasted this task here (I thought I've already remove it from branch) sorry for that!